### PR TITLE
Email Verification Protos

### PR DESCRIPTION
**Pull request by anjalikumari2207**
### Description
- added new protos for `Email Verification` Stage
- Figma -https://www.figma.com/design/WLK2I0TBbQBpeHHY3ibC2D/Kubair-Bank?node-id=137-659&t=PDxBGulUMPwGZ1cx-4

### CODE REVIEW

**Comment by Jha2508** [comment]
Please take a pull from latest `develop` and move your code to existing frontend service.

File: protos-frontend/frontend/FrontendService.proto [hunk 1]
```diff
@@ -34,5 +36,35 @@ message TriggerPhoneVerificationRequest {
...
 
 message TriggerPhoneVerificationResponse {
-    enums.PhoneNumberVerificationResponseType response_type = 1;
+    enums.VerificationResponseType response_type = 1;
 }
+
+
+
+message GetEmailVerificationScreenRequest {
+    string img_url = 1;
```
Comment (Jha2508) [inline protos-frontend/frontend/FrontendService.proto]: Why are we giving these parameters in the request?

File: protos-frontend/frontend/FrontendService.proto [hunk 2]
```diff
@@ -34,5 +36,35 @@ message TriggerPhoneVerificationRequest {
...
+    string sub_title = 3;
+    string frame = 4;
+    string your_emailid = 5;
+    string your_name = 6;
+    ui.Button next_button = 7;
+}
+
+message GetEmailVerificationScreenResponse {
+    enums.VerificationResponseType response_type = 1;
+    string img_url = 2;
```
Comment (Jha2508) [inline protos-frontend/frontend/FrontendService.proto]: Not needed as we can keep the logo inside the App itself

File: protos-frontend/frontend/FrontendService.proto [hunk 3]
```diff
@@ -34,5 +36,35 @@ message TriggerPhoneVerificationRequest {
...
+    string title = 3;
+    string sub_title = 4;
+    string frame = 5;
```
Comment (Jha2508) [inline protos-frontend/frontend/FrontendService.proto]: what is this?

File: protos-frontend/frontend/FrontendService.proto [hunk 4]
```diff
@@ -34,5 +36,35 @@ message TriggerPhoneVerificationRequest {
...
+    string your_emailid = 6;
```
Comment (Jha2508) [inline protos-frontend/frontend/FrontendService.proto]: what is this?

File: protos-frontend/frontend/FrontendService.proto [hunk 5]
```diff
@@ -34,5 +36,35 @@ message TriggerPhoneVerificationRequest {
...
+    string your_name = 7;
```
Comment (Jha2508) [inline protos-frontend/frontend/FrontendService.proto]: what is this?

File: protos-frontend/frontend/FrontendService.proto [hunk 6]
```diff
@@ -34,5 +36,35 @@ message TriggerPhoneVerificationRequest {
...
+    ui.Button next_button = 8;
+}
+
+message TriggerEmailVerificationRequest {
+    string email_id = 1;
```
Comment (Jha2508) [inline protos-frontend/frontend/FrontendService.proto:57]: you need to send both parameters from User right? else there is no point collecting it in the first place

File: protos-frontend/frontend/FrontendService.proto (hunk 3 above)
Comment (anjalikumari2207) [reply protos-frontend/frontend/FrontendService.proto]: Done

File: protos-frontend/frontend/FrontendService.proto (hunk 5 above)
Comment (anjalikumari2207) [reply protos-frontend/frontend/FrontendService.proto]: Done

File: protos-frontend/frontend/FrontendService.proto (hunk 4 above)
Comment (anjalikumari2207) [reply protos-frontend/frontend/FrontendService.proto]: done

**Review by Jha2508 — APPROVED** [review]

**Pull request merged**